db/schema.rb

```diff
@@ -13,12 +13,26 @@
 
 ActiveRecord::Schema.define(version: 20150307235927) do
 
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "sessions", force: :cascade do |t|
+    t.string   "status"
+    t.date     "date"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
   create_table "topics", force: :cascade do |t|
     t.string   "title"
     t.datetime "created_at",             null: false
     t.datetime "updated_at",             null: false
     t.string   "status"
     t.integer  "votes",      default: 0, null: false
+    t.integer  "session_id"
   end
 
+  add_index "topics", ["session_id"], name: "index_topics_on_session_id", using: :btree
+
+  add_foreign_key "topics", "sessions"
```

Comment:
Hmmm... Any idea where this `session` stuff came from in `db/schema` here? There are no migrations adding it. @ohenrik, is it possible that you had added a migration in a separate `sessions` branch, and then this stuff stuck around when you ran your migration on this branch? (Asking because [you talked about working on sessions stuff](https://github.com/jgorset/lean-coffee/issues/7#issuecomment-94703020) :smile:)